store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/integration/FederatedStoreRecursionIT.java

```diff
@@ -64,15 +63,14 @@
     private Graph proxyToRestServiceFederatedGraph;
 
     @Before
-    public void setUp() throws Exception {
+    public void _setup() throws Exception {
+
         graph.execute(new RemoveGraph.Builder()
                 .graphId(PublicAccessPredefinedFederatedStore.ACCUMULO_GRAPH_WITH_EDGES)
                 .build(), user);
         graph.execute(new RemoveGraph.Builder()
                 .graphId(PublicAccessPredefinedFederatedStore.ACCUMULO_GRAPH_WITH_ENTITIES)
                 .build(), user);
-
-        graph = null;
```

Comment:
This was nulled so it wouldn't be used.
It causes a null pointer now because the @Before annotation is kinda superseding the parent annotation. If you place Logging or system.out you can see this occurring.